build/release.mk

```diff
@@ -81,17 +95,11 @@ else
 	cp $(GOPATH)/bin/darwin_amd64/mattermost $(DIST_PATH)/bin # from cross-compiled bin dir
 	cp $(GOPATH)/bin/darwin_amd64/platform $(DIST_PATH)/bin # from cross-compiled bin dir
 endif
-	@# Strip and prepackage plugins
```

Comment:
@crspeller, @levb, @jwilander -- heads up that, because of plugin signing, we can no longer (easily) ship pre-packaged plugins with just the binaries for the platform in question. Thanks to Ali for spotting this! This is likely to balloon the download size in v5.20 accordingly. Is this an immediate concern?

We may be able to tackle this in an later release by signing the stripped down bundles as well, but this would require extra infrastructure effort for which we don't have mana right now.

Reply:
No immediate concerns from me. If it balloons significantly it may be worth a changelog or release note

Reply:
looks like the size has ballooned by roughly `~200MB`

Binaries with my change:
```
1.5G 16 Jan 19:35 mattermost-enterprise-linux-amd64.tar.gz
1.5G 16 Jan 19:24 mattermost-enterprise-osx-amd64.tar.gz
1.5G 16 Jan 19:32 mattermost-enterprise-windows-amd64.zip
```

Binaries on master:
```
1.3G 16 Jan 21:44 mattermost-enterprise-linux-amd64.tar.gz
1.3G 16 Jan 21:31 mattermost-enterprise-osx-amd64.tar.gz
1.3G 16 Jan 21:40 mattermost-enterprise-windows-amd64.zip
```

Reply:
Not that big of a difference.

Reply:
Ya. I didn't realize our binaries had ballooned so large. Guess this little extra doesn't really make too much of a difference.

Reply:
@ali-farooq0, something doesn't seem right here: https://releases.mattermost.com/5.19.0/mattermost-5.19.0-linux-amd64.tar.gz is only 148M. Can you double check?

Reply:
Follow-up conversation at https://community-release.mattermost.com/core/pl/fewj9sy7uibffmjgbusyccdh3a.

Reply:
Ahh apologies, `web-app/dist` was ballooning up my local binaries, after a bit of cleaning:

On master:
```
149M 17 Jan 14:25 mattermost-enterprise-linux-amd64.tar.gz
148M 17 Jan 14:25 mattermost-enterprise-osx-amd64.tar.gz
147M 17 Jan 14:25 mattermost-enterprise-windows-amd64.zip
```

My branch:
```
341M 17 Jan 14:28 mattermost-enterprise-linux-amd64.tar.gz
341M 17 Jan 14:27 mattermost-enterprise-osx-amd64.tar.gz
341M 17 Jan 14:28 mattermost-enterprise-windows-amd64.zip
```

Reply:
We've proposed some options to @amyblais on community -- let's make a call on Monday to resolve, and then we can plan next steps.

Reply:
Looks like we're back in business.

with my new changes (platform specific prepackaged plugins):
```
151M 22 Jan 09:29 mattermost-enterprise-linux-amd64.tar.gz
149M 22 Jan 09:29 mattermost-enterprise-osx-amd64.tar.gz
148M 22 Jan 09:29 mattermost-enterprise-windows-amd64.zip
```